### PR TITLE
[FEAT] CENABLER-5448 - Update guide performance assessment

### PR DESCRIPTION
**Pull request by Dnikodij**
image

image

image

### CODE REVIEW

File: app/src/components/PerformanceAssessment.vue [hunk 1]
```diff
@@ -92,9 +92,9 @@ export default {
...
       switch (activeTab.value) {
         case 0:
-          return 'Welcome to Performance Assessment';
+          return 'About the Performance Assessment tool';
         case 1:
-          return 'Welcome to Performance Assessment';
+          return 'About the Performance Assessment tool';
         default:
           return 'Welcome';
       }
```
Comment (robraccu) [inline app/src/components/PerformanceAssessment.vue:100]: in general could be better  something like:

const guideTitle = computed(() => {
  const titles = {
    0: 'About the Performance Assessment tool',
    1: 'About the Performance Assessment tool',
  };
  return titles[activeTab.value] || 'Welcome';
});

but what means this code ? can 0 and 1 be different?

**Review by robraccu — COMMENTED** [review]
you have a comment

File: app/src/components/PerformanceAssessment.vue (hunk 1 above)
Comment (Dnikodij) [reply app/src/components/PerformanceAssessment.vue:100]: I am using two different files for the guides for the performance assessment, "From NBS to indicators" and "From indicators to NBS" so they each need a title depending on which tool is selected, right now they have the same title but in the future maybe they wont

**Review by nicolaleo — APPROVED** [review]
ok

File: app/src/components/PerformanceAssessment.vue (hunk 1 above)
Comment (robraccu) [reply app/src/components/PerformanceAssessment.vue:100]: ok...

**Review by robraccu — APPROVED** [review]

**Pull request merged**